textile/features.textile

```diff
@@ -406,7 +433,7 @@ h3(#realtimeclient). RealtimeClient
 ** @(RTC1b)@ @autoConnect@ boolean is true by default. If true, as soon as the client library is instantiated, it will connect to Ably. If false, the client library will wait for an explicit @Connection#connect@ to be called before connecting
 ** @(RTC1c)@ @recover@ string, when set, will attempt to recover the connection state of a previous connection
 ** @(RTC1d)@ @realtimeHost@ string, when set, will modify the realtime endpoint host used by this client library
-** @(RTC1e)@ @environment@ string, when set to a value other than "production", use @[environment]-rest.ably.io@ as the REST endpoint and @[environment]-realtime.ably.io@ as the realtime endpoint. For example, a @RealtimeClient@ with an @environment@ of "sandbox" would use "sandbox-rest.ably.io" as the REST endpoint and @sandbox-realtime.ably.io@ as the realtime endpoint. See "TO3k3":#TO3k3 for constraints.
+** @(RTC1e)@ @environment@ string, when set to a value other than "production", use @[environment].realtime.ably.net@ as the REST endpoint and @[environment].realtime.ably.net@ as the realtime endpoint. See "TO3k3":#TO3k3 for constraints.
```

Comment:
> `(RTC1e)` `environment` string, when set to a value other than "production", use `[environment].realtime.ably.net` as the REST endpoint and `[environment].realtime.ably.net` as the realtime endpoint.

This whole section is for client options that are only required by realtime clients or do something different for realtime vs rest clients -- which no longer applies to `environment`. So you could just remove RTC1e entirely

Reply:
OK, deleted (and RTC1d): https://github.com/ably/specification/pull/213/commits/27a16960b674d44ba91873e22f3203756e7d7900